torch/ao/quantization/fx/_decomposed.py

```diff
@@ -908,27 +822,17 @@ def quantize_per_channel_group_meta(
     return torch.empty_like(input, dtype=dtype)
 
 
-quantized_decomposed_lib.define(
-    "dequantize_per_channel_group(Tensor input, Tensor scales, Tensor? zero_points, int quant_min, "
-    "int quant_max, ScalarType dtype, int group_size, ScalarType output_dtype) -> Tensor"
-)
-
-
-@impl(
-    quantized_decomposed_lib,
-    "dequantize_per_channel_group",
-    "CompositeExplicitAutograd",
-)
+@custom_op(f"{ns}::dequantize_per_channel_group", mutates_args=())
 def dequantize_per_channel_group(
     w_int8: torch.Tensor,
     scales: torch.Tensor,
     zero_points: Optional[torch.Tensor],
     quant_min: int,
     quant_max: int,
     dtype: torch.dtype,
-    group_size: int = 128,
-    output_dtype: torch.dtype = torch.float32,
-):
+    group_size: int,
+    output_dtype: torch.dtype,
```

Comment:
We can't have default args anymore?

Reply:
None of the default args I removed were actually being used: they weren't defined in the operator's schema string